src/app/app-routing.module.ts

```diff
@@ -17,7 +17,7 @@ const routes: Routes = [
 ];
 
 @NgModule({
-  imports: [RouterModule.forRoot(routes)],
+  imports: [RouterModule.forRoot(routes, { relativeLinkResolution: 'legacy' })],
```

Comment:
question: why do we need the `legacy` stuff?

Just wanna make sure the lib works without it.